unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/DebugController/DebugBridge.cs

```diff
@@ -131,57 +131,6 @@ public void DumpRendererLockersInfo()
             Debug.unityLogger.logEnabled = originalLoggingValue;
         }
 
-        public void CrashPayloadRequest()
-        {
-            bool originalLoggingValue = Debug.unityLogger.logEnabled;
-            Debug.unityLogger.logEnabled = true;
-
-            var crashPayload = CrashPayloadUtils.ComputePayload
-            (
-                DCL.Environment.i.world.state.GetLoadedScenes(),
-                debugController.GetTrackedMovements(),
-                debugController.GetTrackedTeleportPositions()
-            );
-
-            CrashPayloadResponse(crashPayload);
-
-            Debug.unityLogger.logEnabled = originalLoggingValue;
-        }
-
-        public void CrashPayloadResponse(CrashPayload payload)
-        {
-            string json = JsonConvert.SerializeObject(payload);
-            WebInterface.MessageFromEngine("CrashPayloadResponse", json);
-        }
-
-        [ContextMenu("Dump Crash Payload")]
-        public void DumpCrashPayload()
-        {
-            bool originalLoggingValue = Debug.unityLogger.logEnabled;
-            Debug.unityLogger.logEnabled = true;
-
-            debugLogger.Log(
-                $"MEMORY -- total {Profiler.GetTotalAllocatedMemoryLong()} ... used by mono {Profiler.GetMonoUsedSizeLong()}");
-
-            var payload = CrashPayloadUtils.ComputePayload
-            (
-                DCL.Environment.i.world.state.GetLoadedScenes(),
-                debugController.GetTrackedMovements(),
-                debugController.GetTrackedTeleportPositions()
-            );
-
-            foreach (var field in payload.fields)
-            {
-                string dump = JsonConvert.SerializeObject(field.Value);
-                debugLogger.Log($"Crash payload ({field.Key}): {dump}");
-            }
-
-            string fullDump = JsonConvert.SerializeObject(payload);
-            debugLogger.Log($"Full crash payload size: {fullDump.Length}");
-
-            Debug.unityLogger.logEnabled = originalLoggingValue;
-        }
```

Comment:
is it OK to remove this?

Reply:
Hmm I think I removed this by mistake, good catch